api/envoy/config/route/v3/route_components.proto

```diff
@@ -1385,6 +1385,11 @@ message RateLimit {
 
       // The key to use in the descriptor entry.
       string descriptor_key = 2 [(validate.rules).string = {min_bytes: 1}];
+
+      // If set to true, Envoy skips the descriptor while calling rate limiting service
+      // when header is not present in the request. Defaults to false, which skips calling
```

Comment:
```suggestion
      // when header is not present in the request. By default this skips calling
```
i.e. it's redundant to say that the default of a primitive proto3 type is false, but your description of the actual behavior is very helpful.

Reply:
Sounds good, @htuch! Pushed the changes as you suggested! Thanks!